js/app.js

```diff
@@ -1 +1,112 @@
-console.log('Hello frontend');
+let isGameActive = true;
+
+
+document.addEventListener('DOMContentLoaded', () => {
+   
+    let squares = document.querySelectorAll('.grid-container div')
+    let playerDisplay = document.querySelector('#player')  
+   
+    
+
+    
+
+    let currentPlayer = 'Player X'
+    squares.forEach(square => {
+        square.addEventListener('click', clickOutcome)
+    });
+
+
+    function clickOutcome(e) {
+        const squareArray = Array.from(squares)
+        const index = squareArray.indexOf(e.target)
+        playerDisplay.textContent = "Your turn, " + currentPlayer
+        
+
+        if(currentPlayer === 'Player X'){
+            squares[index].classList.add('playerO')
+            currentPlayer = 'Player O'
+        } else {
+            currentPlayer = 'Player X'
+            squares[index].classList.add('playerX')
+        }
+    function clickOnce(square) {
+        square.target.removeEventlistener('clikc', clickOnece)
+    }
```

Comment:
Alice - this would have been a good start to implementing logic to prevent a user from clicking a square a second time.  Note that you misspelled click and clickOnce at the end of line 33.  You did not call this function anywhere in your code though, so it is not actually being used at all.  Does that make sense?

You could have done something similar with deactivating all of the squares once a player wins.